hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieSparkSqlWriter.scala

```diff
@@ -1035,19 +1039,6 @@ class HoodieSparkSqlWriterInternal {
       log.info(s"Is Async Compaction Enabled ? $asyncCompactionEnabled")
       (commitSuccess && metaSyncSuccess, compactionInstant, clusteringInstant)
     } else {
-      log.error(s"${tableInstantInfo.operation} failed with errors")
```

Comment:
Note to reviewer:
Essentially, these lines were moved inside the SparkDataSourceWriteStatusHandlerCallback